docs/2101UserGuide.md

```diff
@@ -321,69 +324,104 @@ List of commands that can be undone:
 
 #### Redoing a command: `redo`
 
-Redoes a command that the user has undone previously.
+Accidentally used `undo`? Don’t worry, you can redo your undone commands to retrieve your previous data too!
 
-Format: `redo`
+Here’s how you can use it: `redo`
 
-<hr class="command-separator">
-
-#### Clear all Members and Events: `clear`
-Clears all Member and Event entries from CCACommander.
+Here's a list of commands you can redo:
+* `clear`
+* `createMember`
+* `deleteMember`
+* `editMember`
+* `createEvent`
+* `deleteEvent`
+* `editEvent`
+* `enrol`
+* `unenrol`
+* `editEnrolment`
 
-Format: `clear`
 
 <hr class="command-separator">
 
-#### Help: `help`
+#### Clear all Members and Events: `clear`
+Is it time for a new academic year when you need to remove all your previous members and events for new members and 
+events? You can use the `clear` command to instantly remove all your members and events without needing to delete them one by one.
 
-Displays a pop-out window that shows a link to this User Guide.
+Here’s how you can use it: `clear`
 
-Format: `help`
+<div markdown="block" class="alert alert-primary">:bulb: Accidentally cleared your list? 
+Worry not as you can use the `undo` command to retrieve your data! 
```

Comment:
```suggestion
Worry not as you can use the `undo` command to recover your data! 
```
Does recover sound better?

Reply:
yes agree 
![image](https://github.com/AY2324S1-CS2103T-F11-1/tp/assets/99804772/5887a783-9467-417b-9201-c4fb92aec8e4)

Reply:
yes rceover is good!